css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,51 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "DMS Sans", sans-serif;
+}
+
+.container {
+  max-width: 2000px;
+  margin: 0 auto;
+  padding-left: 3rem;
+}
+
+.row {
```

Comment:
We want a little bit of spacing between the `.column-fifth` classes. That can be done by:

```
.row {
  display: flex;
  justify-content: space-between;
}

.column-fifth {
  flex-basis: 19%; // all the columns add up 95% meaning that extra 5% space will be evenly distributed between the columns.
}
```